.github/workflows/step-run-tests.yml

```
@@ -55,6 +55,14 @@ jobs:
         sudo ln -s /usr/lib/x86_64-linux-gnu/libenchant-2.so.2 /usr/lib/libenchant.so.1
         sudo localedef -i ${{ inputs.locale }} -c -f UTF-8 -A /usr/share/locale/locale.alias ${{ inputs.locale }}.UTF-8
 
+        mkdir -p ~/.1cv8/1C/1cv8/conf/
+        echo DisableUnsafeActionProtection=.*> ~/.1cv8/1C/1cv8/conf/conf.cfg
+    - name: Подготовка окружения (Windows)
+      if: runner.os == 'Windows'
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1001:info:1:24: This \1 will be a regular '1' in this context

(shellcheck)